### DISCUSSION TITLE
Welcome to sticky-parallax-header Discussions!

### DISCUSSION THREAD

**Discussion by mateusz1913** [netguru/sticky-parallax-header#315] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by newbie0510** [comment]
hi how to avoid header bouncing when scroll the header. the header bounce whenever i release.

<TabbedHeaderPager
        containerStyle={screenStyles.stretchContainer}
        tabsContainerBackgroundColor={colors.black}
        parallaxHeight={scrollHeight}
        renderHeader={() => {
          return (
            <View pointerEvents="box-none" style={{ height: scrollHeight }}>
              <Text>asd</Text>
            </View>
          );
        }}
        decelerationRate={'normal'}
        bounces={false}
